app/main.py

```diff
@@ -2,0 +1,20 @@
+import datetime
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    num_maskless = 0
+    for friend in friends:
+        if "vaccine" not in friend:
+            return "All friends should be vaccinated"
+
+        expiration_date = friend["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date.today():
+            return "All friends should be vaccinated"
+
+        if not friend.get("wearing_a_mask", False):
+            num_maskless += 1
```

Comment:
This is a minor code efficiency issue, but it's better to avoid comparing a variable to `0` if there is no need. Instead of `if not friend.get('wearing_a_mask', False):`, you can use `if friend.get('wearing_a_mask', True):`. This way, you avoid the need to use the `not` operator.